lib/node_modules/@stdlib/math/base/ops/cnegf/README.md

```diff
@@ -0,0 +1,277 @@
+
+var z1 = new Complex64( -4.0, 5.0 );
+
+var out = cnegf( z1 );
+// returns <Complex64>
+
+var re = realf( out );
+// returns 4.0
+
+var im = imagf( out );
+// returns -5.0
+
+var z2 = new Complex64( 0.0, 0.0 );
+
+out = cnegf( z2 );
+// returns <Complex64>
+
+re = realf( out );
+// returns -0.0
+
+im = imagf( out );
+// returns -0.0
+
+var z3 = new Complex64( NaN, NaN );
+
+out = cnegf( z3 );
+// returns <Complex64>
+
+re = realf( out );
+// returns NaN
+
+im = imagf( out );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var Complex64 = require( '@stdlib/complex/float32' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
+var cnegf = require( '@stdlib/math/base/ops/cnegf' );
+
+function randomComplex() {
+    var re = discreteUniform( -50, 50 );
+    var im = discreteUniform( -50, 50 );
+    return new Complex64( re, im );
+}
+
+var z;
+var o;
+var i;
+
+for ( i = 0; i < 100; i++ ) {
+    z = randomComplex();
+    o = cnegf( z );
+    console.log( 'negate(%s) = %s', z.toString(), o.toString() );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/ops/cnegf.h"
+```
+
+#### stdlib_base_cnegf( z )
+
+Negates a single-precision complex floating-point number.
+
+```c
+#include "stdlib/complex/float32.h"
+#include "stdlib/complex/realf.h"
+#include "stdlib/complex/imagf.h"
+
+stdlib_complex64_t z = stdlib_complex64( 3.0, -2.0 );
+
+stdlib_complex64_t out = stdlib_base_cnegf( z );
+
+double re = stdlib_realf( out );
+// returns -3.0
+
+double im = stdlib_imagf( out );
+// returns 2.0
+```
+
+The function accepts the following arguments:
+
+-   **z**: `[in] stdlib_complex64_t` input value.
+
+```c
+stdlib_complex64_t stdlib_base_cnegf( const stdlib_complex64_t z );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/ops/cnegf.h"
+#include "stdlib/complex/float32.h"
+#include "stdlib/complex/reimf.h"
+#include <stdio.h>
+
+int main() {
+    stdlib_complex64_t x[] = {
```

Comment:
```suggestion
    const stdlib_complex64_t x[] = {
```

We upgraded `cppcheck` (run `make install-deps-cppcheck` locally to upgrade) and it now complains whenever an array which is never mutated is not declared with `const`.